cypress/e2e/02-add_patient.cy.js

```diff
@@ -1,5 +1,8 @@
 import { generatePatient, loginHooks, testNumber } from "../support/e2e";
```

Comment:
@fzhao99 can I revert the changes in this file?

Reply:
ahh sorry for not cleaning up after myself correctly! just pushed the reversion

Reply:
omg no worries! wanted to make sure you didn't need these! :) Thank you for the cleanup and testing!!